packages/arb-token-bridge-ui/src/util/wagmi/wagmiAdditionalNetworks.ts

```diff
@@ -1,5 +1,49 @@
 import { Chain } from 'wagmi'
-import { ChainId, rpcURLs } from '../networks'
+import { ChainId, explorerUrls, rpcURLs } from '../networks'
+
+export const sepolia: Chain = {
```

Comment:
We don't get the network from wagmi because the public RPCs are getting rate limited.

Reply:
I think we need to do both `default` and `public`:

https://github.com/wagmi-dev/references/blob/main/packages/chains/src/sepolia.ts#L17